e2e/test/scenarios/embedding/embedding-full-app.cy.spec.js

```diff
@@ -218,10 +218,8 @@ describe("scenarios > embedding > full app", () => {
           url: "/question/" + ORDERS_QUESTION_ID,
           qs: { side_nav: false, logo: false, breadcrumbs: false },
         });
-        cy.findAllByRole("banner")
-          .should("have.length", 1)
-          .findByText("Connect to your database to get the most from Metabase.")
-          .should("exist");
+        cy.findByDisplayValue("Orders");
```

Comment:
Why do we need to change the test logic here and stop testing the db connection banner here?
Also, it seems that these specs should be EE with token

Reply:
The test wasn't really about the top-most banner CTA "Connect to database...". I think @WiNloSt  added that to the test simply because we were always running with the premium token before and that's what you'd see with the token active.

The test simply makes sure that the main header (that contains the logo, breadcrumbs, etc.) doesn't show up if the right query parameters are set.

> Also, it seems that these specs should be EE with token

That is most likely true.
I haven't had the time to check every single feature's requirement and I plan to do that immediately after this overhaul. This spec will have to be re-written anyaway.

tl;dr
Instead of trying to do it all at once, the goal was to make tests in the current form work with the new logic first, and only then fix the correct behavior according to the feature flags.

Reply:
@nemanjaglumac 
> I think @WiNloSt added that to the test simply because we were always running with the premium token before and that's what you'd see with the token active.
>
> The test merely makes sure that the main header (that contains the logo, breadcrumbs, etc.) doesn't show up if the right query parameters are set.

That's very accurate. The point was to test there's no header except the DB prompt banner. For context, originally the test only tested that there was no banner at all using `findByRole("banner", ...)`. It was changed after when the database prompt banner was introduced since the database prompt banner will be found by `findByRole("banner", ...)` too.